site/build/reference/render.js

```diff
@@ -55,14 +58,8 @@ function makeAvailability(value) {
   return availability;
 }
 
-function makeContribute(source) {
-  var base =
-    'https://github.com/briangreenery/relevance.io/tree/master/site/reference/';
-
-  return base + source;
-}
-
 function renderProperty(property, template) {
+  var reference = property.resultType;
```

Comment:
I think if there's an `indexType` and no `directObjectType`, it should be linked to the `indexType`. This would make the link better for properties like `relevant ( bes computer, bes property ) : boolean`. Maybe the `reference` could be created by `associate`? Also, I'd feel safer if we were asserting that the reference wasn't on a tuple type so that we could be sure the link is valid.